resources/views/user/password/reset-password.blade.php

```diff
@@ -1,34 +1,63 @@
 @extends('layouts.app')
 
 @section('content')
-    <div>
-        <div>
-            <h1>Forgot Password Page</h1>
-            <form action="{{ route('password.update') }}" method="post">
-                @csrf
-                <div>
-                    <input type="string" id="token" hidden="hidden" name="token" value="{{ $token }}">
-                    <x-input-error for="token"/>
-                </div>
-                <div>
-                    <label for="email">email:</label>
-                    <input type="email" id="email" readonly="readonly" name="email" value="{{ ($_GET['email']) }}">
-                    <x-input-error for="email"/>
-                </div>
-                <div>
-                    <label for="password">password:</label>
-                    <input type="password" id="password" name="password" value="{{ old('password') }}">
-                    <x-input-error for="password"/>
-                </div>
-                <div>
-                    <label for="confirm_password">password_confirmation :</label>
-                    <input type="password" id="password_confirmation" name="password_confirmation" value="{{ old('password_confirmation') }}">
-                    <x-input-error for="password_confirmation"/>
-                </div>
-                <div>
-                    <button type="submit">Set New Password</button>
-                </div>
-            </form>
+  <div className="container mx-auto">
+    <div class="min-h-full flex flex-col justify-center py-12 sm:px-6 lg:px-8">
+      <div class="mt-8 sm:mx-auto sm:w-full sm:max-w-lg">
+        <div class="bg-white pt-2 pb-8 px-4 shadow sm:rounded-20 sm:px-10">
+          <form class="space-y-6" action="{{ route('password.update') }}" method="post">
+            @csrf
+            <div>
+              <h1 class="text-xl">Reset Password</h1>
+            </div>
+            <div>
+              <input type="string" id="token" hidden="hidden" name="token" value="{{ $token }}">
+              <x-input-error for="token" />
+            </div>
+            <div>
+              <label for="email" class="block text-sm font-medium text-gray-700">
+                Email
+              </label>
+              <div class="mt-1">
+                <input id="email" name="email" type="email" placeholder="Email..." value="{{ ($_GET['email']) }}" required
```

Comment:
Placeholder should be used to show user how input should looks like.
in this case you can write `example@example.com`
The same for the rest placeholders in this PR.

Reply:
I'm not sure that's a common rule.

Reply:
From my point of view it will increase user experience.